cypress/e2e/albums.cy.js

```diff
@@ -128,21 +128,21 @@ describe('Manage albums', () => {
 		cy.contains('Save').click()
 	})
 
-	it('Edit an album\'s location', () => {
+	it('Edit an album\'s place', () => {
 		cy.get('[aria-label="Open actions menu"]').click()
 		cy.contains('Edit album details').click()
-		cy.get('form [name="location"]').clear().type('New location')
+		cy.get('form [name="place"]').clear().type('New place')
 		cy.contains('Save').click()
 
-		cy.contains('New location')
+		cy.contains('New place')
 
 		cy.reload()
 
-		cy.contains('New location')
+		cy.contains('New place')
```

Comment:
It seems a bit too broad. We can merge but I would make sure to check against the specific section that contains this string, not the entire page

Reply:
Was indeed too broad :)
https://github.com/nextcloud/photos/pull/1657